go/vt/vttablet/tabletmanager/rpc_backup.go

```diff
@@ -83,6 +83,11 @@ func (agent *ActionAgent) Backup(ctx context.Context, concurrency int, logger lo
 			return err
 		}
 		originalType = tablet.Type
+		if originalType == topodatapb.TabletType_MASTER {
```

Comment:
Doesn't the `refreshTablet` call cause the the VREngine to shutdown? I think that's the right way for this to happen.

Reply:
It doesn't right now, but I agree that is a better place to do this.

Reply:
oh that's right. This is an old issue and I didn't realize it had been fixed.